make the readme product-agnostic and remove mentions to github actions (#2)
sonarsource-cfamily-examples/macos-xcode-coverage-gh-actions-sc: README.adoc

> = C++ example project for code coverage with Xcode

Reviewer: ```suggestion
= C++ example project for generating test coverage with Xcode
```

image:{img-build-status}[Build Status, link={uri-build-status}]
image:{img-qg-status}[Quality Gate Status,link={uri-qg-status}]

> *The test coverage shown here works on Both SonarQube and SonarCloud. This project is analyzed on SonarCloud https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_macos-xcode-coverage-gh-actions-sc[SonarCloud]!*

Reviewer: ```suggestion
*The test coverage shown here works on both SonarQube and SonarCloud. This project is analyzed on https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_macos-xcode-coverage-gh-actions-sc[SonarCloud]!*
```

:note-caption: :information_source:
NOTE: This simple example aims at demonstrating code coverage in action. It explains how to add coverage on a project for which the analysis is already configured. Analysis configuration examples are available https://github.com/sonarsource-cfamily-examples?q=xcode[here]. +

> Adding code coverage is done the same way regardless of using the Build Wrapper or a Compilation Database to cconfigure the analysis.

Reviewer: ```suggestion
Adding code coverage is done the same way regardless of whether you use the Build Wrapper or a Compilation Database to configure the analysis.
```

> After your build and before your running `sonar-scanner`, proceed with the following steps:

Reviewer: ```suggestion
After your build and before running `sonar-scanner`, proceed with the following steps:
```

. Run the test binaries with `-enableCodeCoverage YES` to count covered lines

> . Run link:xccov-to-generic.sh[xccov-to-generic.sh] on the `*.xcresult` files to produce a https://docs.sonarcloud.io/enriching/test-coverage/generic-test-data/[Generic Test Data] format that is compatible with SonarCloud

Reviewer: ```suggestion
. Run link:xccov-to-generic.sh[xccov-to-generic.sh] on the `*.xcresult` files to produce a https://docs.sonarcloud.io/enriching/test-coverage/generic-test-data/[Generic Test Data] format that is compatible with SonarCloud and SonarQube
```

. Run `sonar-scanner` as the final step

> * Point the sonar.coverageReportPaths property to the generated coverage-report file

Reviewer: ```suggestion
* Point the `sonar.coverageReportPaths` property to the generated coverage-report file
```
